src/devices/Amg88xx/samples/Amg88xx.Sample.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Amg88xx.Samples
+{
+    /// <summary>
```

Comment:
Thanks as well for the very detailed sample

Reply:
You're welcome!